sdk/python/feast/loaders/ingest.py

```diff
@@ -41,8 +43,11 @@ def _encode_pa_tables(file: str, fs: FeatureSet, row_group_idx: int) -> List[byt
             File directory of all the parquet file to encode.
             Parquet file must have more than one row group.
 
-        fs (feast.feature_set.FeatureSet):
-            FeatureSet describing parquet files.
+        feature_set (str):
+            FeatureSet describing the full feature set refference in the format f"{project}/{name}:{version}".
+
+        fields (dict[str, enum.Enum.ValueType]):
+            Fields represent dictionary of field name and field value type.
```

Comment:
Fields don't represent a dictionary, they're _represented by_ a dictionary. The type tells us this though, without prose. It is helpful to know what the `str` key is though. I'd suggest perhaps:

> A mapping of field names to their value types.

This is a pretty painstakingly documented private helper function 😄